src/store/model-store.js

```diff
@@ -4,11 +4,24 @@ import { createAddress, readAddress } from 'consus-core/identifiers';
 let models = [
     {
         address: 'm8y7nEtAe',
-        name: 'Resistor'
+        name: 'Resistor',
+        description: 'V = IR',
+        location: 'Shelf 14',
+        isFaulty: false,
+        faultDescription: '',
+        price: '$10.50',
```

Comment:
`price` should be a number instead of a string because parsing strings is excessive work.